.github/workflows/buildAndTest.yml

```diff
@@ -44,9 +47,37 @@ jobs:
       with:
         cache-suffix: ${{ matrix.os }}-${{ matrix.targetarch }}-${{ matrix.llvmtype }}-${{ matrix.llvmbuildtype }}
 
+    - name: Configure llvm-cross-compile
+      # libzstd on GH Runners are only x86_64 to remove them.
+      if: matrix.targetarch == 'arm64'
+      run: |
+        sudo rm -rf /usr/local/lib/libzstd.*.dylib
```

Comment:
Would -DLLVM_ENABLE_ZSTD =OFF below work also?

Reply:
Ah yes. I was copying what we were doing in IREE. 😀 I think ideally llvm upstream should check for architecture along with presence before linking. I know conda also adds it to the path inadvertently.

Reply:
+1 on using the flag if it works.